### PR TITLE
Added the RekognitionFacesManagementPolicy for a full faces managemen…

### PR DESCRIPTION
**Pull request by HCans**
…t within a Collection in AWS Rekognition

*Issue #, if available:*

*Description of changes:*
Hello. I am adding the "RekognitionFacesManagementPolicy" template policy since I need the rekognition:DeleteFaces permission for an application that I want to publish in the repository. 

I think a template like this could cover the most user cases when managing faces.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by keetonian** [comment]
Hi! In my initial look through this template, the scoping of resources as well as the policy actions you've defined look appropriate. Could you also add this policy to the tests like was done in https://github.com/awslabs/serverless-application-model/pull/586/files? Thanks!

**Comment by HCans** [comment]
Done! 👍

File: tests/translator/input/all_policy_templates.yaml [hunk 1]
```diff
@@ -98,6 +98,9 @@ Resources:
 
         - CloudWatchDashboardPolicy: {}
 
+        - RekognitionFacesManagementPolicy:
```
Comment (keetonian) [inline tests/translator/input/all_policy_templates.yaml]: Adding this policy to the end of the list instead of the middle should fix the failing tests.

**Comment by HCans** [comment]
Any idea if there is a release planned this week that would include this template? I am asking because I would like to publish for the devPost Hackaton. Thank you!

**Comment by brettstack** [comment]
@HCans I can't comment on dates, but we'll get this into the 1.8.0 release keeping in mind that this is for the hackathon.

**Comment by keetonian** [comment]
This feature has now been released

**Pull request merged**